CMakeLists.txt

```
@@ -57,9 +57,16 @@ option(ROBOTOLOGY_ENABLE_CORE "Enable compilation of core software libraries." T
 option(ROBOTOLOGY_ENABLE_DYNAMICS "Enable compilation of software for balancing and walking." FALSE)
 option(ROBOTOLOGY_ENABLE_TELEOPERATION "Enable compilation of software for teleoperation." FALSE)
 option(ROBOTOLOGY_ENABLE_ICUB_HEAD "Enable compilation of software necessary on the system running in the head of the iCub robot." FALSE)
+option(ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS "Enable compilation of software for human dynamics estimation." FALSE)
 include(CMakeDependentOption)
-cmake_dependent_option(ROBOTOLOGY_USES_CFW2CAN "Enable compilation of software that runs on the head of the iCub and depends on CFW2CAN." FALSE "ROBOTOLOGY_ENABLE_ICUB_HEAD" FALSE)
 
+option(ROBOTOLOGY_ESDCAN_OPTION "Enable ESDCAN option" FALSE)
```

```suggestion
set(ROBOTOLOGY_ESDCAN_OPTION FALSE)
```

Let's have this as a "normal" CMake variable, not a cache cmake variable as otherwise we may have some tricky issue (and having the option in the list of user options could confused users).
Alternatively we can also always add the `ROBOTOLOGY_USES_ESDCAN` option without any dependent option, like we do we other `ROBOTOLOGY_USES_***` option.

---

I think it is better to have dependency, since without those dependencies, this option does not provide anything.

---

Ack.